lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -2,17 +2,35 @@
 Tests for users API
 """
 import datetime
+import ddt
+from django.conf import settings
 from django.utils import timezone
+from django.template import defaultfilters
+from mock import patch
+import pytz
 
 from xmodule.modulestore.tests.factories import ItemFactory, CourseFactory
 from student.models import CourseEnrollment
 from certificates.models import CertificateStatuses
 from certificates.tests.factories import GeneratedCertificateFactory
+from courseware.access_response import (
+    MilestoneError,
+    StartDateError,
+    VisibilityError,
+)
+from xmodule.course_module import DEFAULT_START_DATE
 
 from .. import errors
 from ..testutils import MobileAPITestCase, MobileAuthTestMixin, MobileAuthUserTestMixin, MobileCourseAccessTestMixin
 from .serializers import CourseEnrollmentSerializer
 
+from util.milestones_helpers import (
+    set_prerequisite_courses,
+    seed_milestone_relationship_types,
+)
+
+# pylint: disable=no-member
```

Comment:
In that case, please
* Add a comment clarifying this
* Remove the other `disable=no-member`s in this file
* If it works, change to the line to `# pylint: disable=E1103` which only disables `no-member` warnings in the case that types couldn't be inferred, which seems to be the case here. That way, this won't disable `no-member` warnings for which pylint actually knows what the types are (see http://pylint-messages.wikidot.com/messages:e1103).